sdk/ml/azure-ai-ml/azure/ai/ml/_restclient/v2023_08_01_preview/aio/operations/_operations.py

```diff
@@ -48,21 +55,20 @@ def __init__(self, client, config, serializer, deserializer) -> None:
     def list(
         self,
         **kwargs: Any
-    ) -> AsyncIterable["_models.AmlOperationListResult"]:
+    ) -> AsyncIterable["_models.OperationListResult"]:
```

Comment:
Is this a breaking change?

Reply:
As far as I can tell, there are two changes:
`OperationListResult` returns things of `OperationResult`, which all have the same properties as `AmlOperation` except `OperationResult` also additionally have an enum field `actionType` for marking internal only APIs. `OperationListResult` also has a `nextLink` field that contains a URL for getting the next set of operation list results. I'm not entirely sure how Preview APIs work, especially because this _was_ from the Azure Rest API specs Aug 2023 Preview development branch, but that has now been merged into `main`, with this file being the final result. So does the client before no longer work/match the backend contract?

Reply:
@annatisch do you know if this is breaking and if yes, do you know if it is intentional?

Reply:
If would come down to how this is exposed in the public API - these generated operations aren't exposed directly.
So some questions:
- which public API is this operation working underneath, for example it looks like `MLClient.workspaces.list()`?
- Are we changing the contract of that API at all. If not, this is not a problem.
- If we are hoping to change this API contract, ss this public API declared `experimental` or not.

Reply:
This new API view seems to only be consumed by `FeatureSetOperations`, which is marked as experimental anyway. So I'd say this is not an issue for now.